app/assets/stylesheets/application.scss

```diff
@@ -607,30 +609,40 @@ label.remove_time_span {
 }
 
 .documentHeader {
-  position: relative;
-  h3 {
-    font-size: 100%;
-  }
+  clear: both;
+}
+// Ridiculous specificity to override Blacklight
```

Comment:
I'll try to take a look, but how much of Blacklight's specificity is to override Bootstrap? 😨